packages/gce-disk-expand/src/usr/share/initramfs-tools/hooks/expand-rootfs

```diff
@@ -36,3 +36,4 @@ copy_exec /sbin/e2fsck /bin
 copy_exec /sbin/blkid /bin
 copy_exec /sbin/resize2fs /bin
 copy_exec /bin/udevadm /bin
+copy_exec /usr/sbin/dmidecode /bin
```

Comment:
What's dmidecode used for?

Reply:
parted i think calls it and complains without it

Reply:
Oh yes I think you are right.